pysparkling/rdd.py

```diff
@@ -2080,10 +2080,9 @@ def toDF(self, schema=None, sampleRatio=None):
         >>> rdd.toDF().collect()
         [Row(age=1, name='Alice')]
         """
-        # Top level import would cause cyclic dependencies
-        # pylint: disable=import-outside-toplevel
-        from pysparkling import Context
-        from pysparkling.sql.session import SparkSession
+        # pylint: disable=import-outside-toplevel  # against a cyclic-import
+        from .sql.session import SparkSession
+
         sparkSession = SparkSession._instantiatedSession or SparkSession(Context())
```

Comment:
I don't know why a new context is created here. The import issues probably come from here. There is a `self.context` that you get without importing `Context`. If you can change those, can you avoid importing Context at the top of the file?

Reply:
Thanks for your help! I fixed it as such and now it works fine.